db/migrate/20171031213410_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :username, null: false
+      t.string :password_digest, null: false
+      t.string :email, null: false
```

Comment:
Probably doesn't belong here as a comment, but do you know how to validate email? Like, check for an @ sign or something?